civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -115,6 +115,7 @@
 
 @import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/variables";
 @import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/mixins/tables";
+@import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/mixins/utility";
```

Comment:
I would call it `utilities`. What do you think?

Reply:
Yes its a better name, changed the same.

Reply:
Not required anymore. As it is moved to shoreditch